package.xml

```diff
@@ -112,7 +112,7 @@
         <required>
             <php>
                 <min>5.6</min>
-                <max>7.2.99</max>
```

Comment:
since we're not yet do much testing with 7.3, I'm a bit wary of advertising the support for it :D  . WDYT ?

Reply:
I'm totally cool with holding off until we feel more tested for sure! :)